docs/89_glossary.md

```diff
@@ -82,29 +79,32 @@ county clerk and recorder who possesses RLA administrative user
 privileges sufficient to upload a cast vote record and ballot manifest
 for the county.
 
-* **contest name** - The title of a ballot contest.
+* **contest name** - The title of a contest.
 
-* **election day** - The day mandated by Colorado law for conducting a
+* **election day** - The final day on which voters can cast a ballot in a
 State Primary Election, Presidential Primary Election, Coordinated
-Election, and General Election.
+Election, or General Election.
 
 * **offeror** - A vendor that submits a responsible bid for this
 Documented Quote.
 
 * **pseudo-random number generator** - A random number generator
 application that is further explained at
 http://statistics.berkeley.edu/~stark/Java/Html/sha256Rand.htm
+Test data is available at https://github.com/cjerdonek/rivest-sampler-tests
 
-* **random seed** - A random seed (or seed state, or just seed) is a
-number (or vector) used to initialize a pseudorandom number generator.
+* **random seed** - A random seed (or seed state, or just seed) is data, such
+as a number, vector or string, used to initialize a pseudorandom number generator.
 
 * **responsible bid** - A bid from a vendor that can responsibly
 (i.e. is reasonably able and qualified) do the work stated in the
 solicitation.
 
-* **risk-limiting audit (RLA)** - An audit that provides strong statistical
-evidence that the election outcome is right, and has a high
-probability of correcting a wrong outcome. One paper describing
+* **risk-limiting audit (RLA)** - a procedure for manually checking a
+sample of ballots (or voter-verifiable records) that is guaranteed to have
+a large, pre-specified chance of correcting the reported outcome if the
+reported outcome is wrong. (An outcome is wrong if it disagrees with the
```

Comment:
This parenthetical definition of "wrong outcome" is probably not needed, since it is defined below, and the definition is intuitive enough that it probably doesn't need to be restated to roughly understand this definition.